circle.yml

```diff
@@ -0,0 +1,21 @@
+machine:
+  xcode:
+    version: 8.2
+
+checkout:
+  post:
+    - git submodule update --init --recursive
+
+dependencies:
+  pre:
+    - bundle install
+    - gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+    - Scripts/install_swiftlint.sh
```

Comment:
Shouldn't that be a rake task? Or is that in the other PR splitting tasks in rakelib? (If so how do we join-merge those?)